### PR TITLE
Update from EventEmitter to CyEventEmitter (+cypress@9.7.0)

### PR DESCRIPTION
**Pull request by merrywhether**
Cypress types were updated in a [PR included in the 9.7.0 release](https://github.com/cypress-io/cypress/pull/21286/files), changing `EventEmitter` to `CyEventEmitter`. This PR updates to 9.7.0 and fixes the types accordingly.

It also winds up updating `package-lock.json` from v1 to v2 as the latest npm longer supports v1. This update is forced by simply running `npm i` with `npm@8.5.0`. I'm not sure how to work around this will updating the Cypress dependency version and keeping the lockfile in sync, but I imagine other contributors would run into this as well. Let me know if this is a problem.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,10 +26,10 @@
   "license": "MIT",
   "devDependencies": {
     "@types/node": "14.14.22",
-    "cypress": "9.6.1",
+    "cypress": "9.7.0",
     "prettier": "2.2.1",
-    "typescript": "4.1.3",
-    "semantic-release": "19.0.2"
+    "semantic-release": "19.0.2",
+    "typescript": "4.1.3"
```
Comment (merrywhether) [inline package.json:32]: `npm i` alphabetized these

**Comment by peterkrieg** [comment]
Thanks for putting this together @merrywhether .  I was having the same problem.  I tested this out and the minor types change seems to also work with cypress 10 👍

**Comment by dvargas92495** [comment]
@bahmutov anything we could do to help get this PR over the finish line? Bug is affecting me too on cypress 10.1

**Comment by peterkrieg** [comment]
In case it helps someone else until this PR ships, I have a hosted version of the fixes, and you just need to put the following in your `package.json` file:

```json
"local-cypress": "https://github.com/peterkrieg/local-cypress/archive/refs/tags/v1.2.7.tar.gz",
```

**Comment by bahmutov** [comment]
Super, let me merge and release

**Pull request merged**